examples/tutorials/DeepChemXAlphafold.ipynb

```diff
@@ -0,0 +1,1767 @@
+{
```

Comment:
<p>Is there a tool we can use to visualize pockets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
nglview is commonly used to visualize proteins. However, it does't work properly on google colab. https://github.com/googlecolab/colabtools/issues/2853#issuecomment-1171699299. An issue was raised on it and google colab team has triaged it. We might have to look for alternatives of protein visualization on google colab.